### DISCUSSION TITLE
Using the 'abbreviations' YAML option: how-to ?

### DISCUSSION THREAD

**Discussion by ma-riviere** [quarto-dev/quarto-cli#2483] [Q&A] [answered]
Hi,

I've been perusing the docs, but I couldn't find any example on how to use the [`abbreviations` YAML option](https://quarto.org/docs/reference/formats/html.html#rendering). From what I understand, you can define a file containing a list of abbreviations, and it will match any instance of the defined abbreviations in the text (with an `<abbr>` HTML tag ?)

What file format should I use, and what syntax is used to define abbreviations within this file ?

Thank you

**Comment by dragonstyle** [comment, marked as answer]
The abbreviations file is really only used when reading markdown (emitting abbreviated text for PDF generation). This is really the underlying `abbreviations` options from pandoc which is described as:


> Specifies a custom abbreviations file, with abbreviations one to a line. If this option is not specified, pandoc will read the data file abbreviations from the user data directory or fall back on a system default. To see the system default, use pandoc --print-default-data-file=abbreviations. The only use pandoc makes of this list is in the Markdown reader. Strings found in this list will be followed by a nonbreaking space, and the period will not produce sentence-ending space in formats like LaTeX. The strings may not contain spaces.

You can see the default list being used by Quarto/Pandoc using `quarto pandoc --print-default-data-file=abbreviations`.

**Reply by ma-riviere** [reply]
Ah, I see, thank you ! 
I thought it would convert all parsed acronyms / abbreviations matching the list provided into <abbr> HTML tags.

Is there any existing option / filter / extension to do that ?

**Reply by dragonstyle** [reply]
I have not yet seen one that does this, though it wouldn't be too bad to write a filter that accomplishes this.

**Reply by ma-riviere** [reply]
Thank you :)
